client/src/productDetails/components/AddToCart.jsx

```diff
@@ -15,42 +14,91 @@ const AddBagAndFavorite = styled.div`
 `;
 
 const AddtoBagBtn = styled.button`
-  border: 1px solid black;
-  background-color: #363636;
-  color: #F8F8F8;
-  cursor: pointer;
-  padding: 15px 32px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  transition-duration: 0.4s;
+  transition-duration: 0.3s;
+  width: 40%;
+  float: left;
+  margin: 0 10px;
   &:hover {
-    background-color: #F8F8F8;
+    font-weight: bold;
     color: #363636;
-    text-decoration: bold;
+    border: 2px solid black;
   }
 `;
 
 const FavoriteBtn = styled.button`
-  border: 1px solid black;
-  background-color: #363636;
-  color: #F8F8F8;
-  cursor: pointer;
-  padding: 15px 32px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  transition-duration: 0.4s;
+  transition-duration: 0.3s;
+  width: 20%;
+  float: left;
+  margin: 0 10px;
   &:hover {
-    background-color: #F8F8F8;
     color: #DC2F2F;
-    text-decoration: bold;
+    font-weight: bold;
+    border: 2px solid black;
+  }
+`;
+
+const SizeSelector = styled.div`
+  width: 40%;
+  float: left;
+  padding: 10px;
+`;
+
+const QuantitySelector = styled.div`
+  width: 40%;
+  float: left;
+  padding: 10px;
+`;
+
+const CustomSelector = styled.select`
+  box-sizing: border-box;
```

Comment:
You might not need this line anymore, not sure

Reply:
Thank you!